### ISSUE TITLE
update-properties: resolve new versions in parallel

### ISSUE THREAD

**Issue by delanym** [mojohaus/versions#1157]
Running a goal like `versions:update-properties -Dincludes=*:*` takes for ages with many dependencies.
Please allow resolving these in parallel like Maven would do itself.

**Comment by andrzejj0** [comment]
PRs are welcome.

**Comment by andrzejj0** [comment]
> @andrzejj0  - Does this #1207 resolve it?

I will check when I have time.

**Comment by andrzejj0** [comment]
No, #1207 does not resolve this. That PR updates UseLatestVersionVersionsMojoBase where it parallelises the process of finding updates for versions; property updates use different logic.
